src/shared/components/VideoPlayer/videoJsPlayer.ts

```diff
@@ -47,23 +49,28 @@ const hotkeysHandler = (event: videojs.KeyboardEvent, playerInstance: VideoJsPla
       playerInstance.currentTime(currentTime + 10)
       return
     case 'ArrowUp':
-      if (currentVolume <= 0.95) {
-        playerInstance.volume(currentVolume + 0.05)
+      if (playerInstance.muted()) {
+        playerInstance.muted(false)
+      }
+      if (currentVolume <= 0.9) {
```

Comment:
this cond can easily break stuff if we change the step at any point

I suggest something like

```
if(currentVolume <= 1) {
   playerInstance.volume(Math.min(currentVolume + VOLUME_STEP, 1))
}
```

wdyt?

Reply:
nice, applied something similar also to `ArrowDown`